### PR TITLE
:sparkles: Source Zendesk Support: add Skills Based Routing streams

### PR DESCRIPTION
**Pull request by davydov-d**
## What
https://github.com/airbytehq/airbyte/issues/16887
Add `Account attributes` and `Attribute definitions` streams

## How
- Add new streams
- Add them to existing unit tests
- Add them to existing CAT
- Bump connector version

### CODE REVIEW

**Comment by octavia-squidington-iii** [comment]
##  source-zendesk-support test report (commit [`72bc42c6ab`](https://github.com/airbytehq/airbyte/pull/27450/commits/72bc42c6ab12c6eb3a5b3ff4dea72c93e3d23e08)) - ✅

⏲️  Total pipeline duration: 912 seconds

| Step                                                                          | Result   |
|:------------------------------------------------------------------------------|:---------|
| Validate airbyte-integrations/connectors/source-zendesk-support/metadata.yaml | ✅        |
| Connector version semver check.                                               | ✅        |
| Connector version increment check.                                            | ✅        |
| QA checks                                                                     | ✅        |
| Connector package install                                                     | ✅        |
| Build source-zendesk-support docker image for platform linux/x86_64           | ✅        |
| Unit tests                                                                    | ✅        |
| Acceptance tests                                                              | ✅        |

🔗 [View the logs here](https://github.com/airbytehq/airbyte/actions/runs/5294313125)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/tools/ci_connector_ops/ci_connector_ops/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=source-zendesk-support test
```

**Comment by octavia-squidington-iii** [comment]
##  source-zendesk-support test report (commit [`f9d0f1eab5`](https://github.com/airbytehq/airbyte/pull/27450/commits/f9d0f1eab5773c5bddacc09034ab7f7d7380a7ad)) - ✅

⏲️  Total pipeline duration: 895 seconds

| Step                                                                          | Result   |
|:------------------------------------------------------------------------------|:---------|
| Validate airbyte-integrations/connectors/source-zendesk-support/metadata.yaml | ✅        |
| Connector version semver check.                                               | ✅        |
| Connector version increment check.                                            | ✅        |
| QA checks                                                                     | ✅        |
| Connector package install                                                     | ✅        |
| Build source-zendesk-support docker image for platform linux/x86_64           | ✅        |
| Unit tests                                                                    | ✅        |
| Acceptance tests                                                              | ✅        |

🔗 [View the logs here](https://github.com/airbytehq/airbyte/actions/runs/5294314405)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/tools/ci_connector_ops/ci_connector_ops/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=source-zendesk-support test
```

File: airbyte-integrations/connectors/source-zendesk-support/acceptance-test-config.yml [hunk 1]
```diff
@@ -31,6 +31,10 @@ acceptance_tests:
           # This stream is only available for enterprise accounts https://developer.zendesk.com/api-reference/ticketing/account-configuration/audit_logs/
           - name: "audit_logs"
             bypass_reason: "no records"
+          - name: "account_attributes"
```
Comment (lazebnyi) [inline airbyte-integrations/connectors/source-zendesk-support/acceptance-test-config.yml:34]: Is it possible to seed those streams?

Comment (davydov-d) [reply airbyte-integrations/connectors/source-zendesk-support/acceptance-test-config.yml]: done

**Review by lazebnyi — APPROVED** [review]

**Comment by octavia-squidington-iii** [comment]
##  source-zendesk-support test report (commit [`3115b7da3f`](https://github.com/airbytehq/airbyte/pull/27450/commits/3115b7da3ff939a31d068bc2664aa341451f926b)) - ❌

⏲️  Total pipeline duration: 923 seconds

| Step                                                                          | Result   |
|:------------------------------------------------------------------------------|:---------|
| Validate airbyte-integrations/connectors/source-zendesk-support/metadata.yaml | ✅        |
| Connector version semver check                                                | ✅        |
| Connector version increment check                                             | ✅        |
| QA checks                                                                     | ✅        |
| Code format checks                                                            | ❌        |
| Connector package install                                                     | ✅        |
| Build source-zendesk-support docker image for platform linux/x86_64           | ✅        |
| Unit tests                                                                    | ✅        |
| Acceptance tests                                                              | ❌        |

🔗 [View the logs here](https://storage.googleapis.com/airbyte-ci-reports/airbyte-ci/connectors/test/pull_request/ddavydov_16887-source-zendesk-support-add-skills-based-routing-streams/1687784130/3115b7da3ff939a31d068bc2664aa341451f926b/source-zendesk-support/0.5.0/output.html)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/tools/ci_connector_ops/ci_connector_ops/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=source-zendesk-support test
```

**Comment by octavia-squidington-iii** [comment]
##  source-zendesk-support test report (commit [`d1496e0259`](https://github.com/airbytehq/airbyte/pull/27450/commits/d1496e02598fa71aaa61adb36e5e7df0a87ff161)) - ❌

⏲️  Total pipeline duration: 907 seconds

| Step                                                                          | Result   |
|:------------------------------------------------------------------------------|:---------|
| Validate airbyte-integrations/connectors/source-zendesk-support/metadata.yaml | ✅        |
| Connector version semver check                                                | ✅        |
| Connector version increment check                                             | ✅        |
| QA checks                                                                     | ✅        |
| Code format checks                                                            | ✅        |
| Connector package install                                                     | ✅        |
| Build source-zendesk-support docker image for platform linux/x86_64           | ✅        |
| Unit tests                                                                    | ✅        |
| Acceptance tests                                                              | ❌        |

🔗 [View the logs here](https://storage.googleapis.com/airbyte-ci-reports/airbyte-ci/connectors/test/pull_request/ddavydov_16887-source-zendesk-support-add-skills-based-routing-streams/1687786126/d1496e02598fa71aaa61adb36e5e7df0a87ff161/source-zendesk-support/0.5.0/output.html)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/tools/ci_connector_ops/ci_connector_ops/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=source-zendesk-support test
```

**Comment by octavia-squidington-iii** [comment]
##  source-zendesk-support test report (commit [`27783b08ac`](https://github.com/airbytehq/airbyte/pull/27450/commits/27783b08acf5af802a6c549701c5113b388802b7)) - ❌

⏲️  Total pipeline duration: 921 seconds

| Step                                                                          | Result   |
|:------------------------------------------------------------------------------|:---------|
| Validate airbyte-integrations/connectors/source-zendesk-support/metadata.yaml | ✅        |
| Connector version semver check                                                | ✅        |
| Connector version increment check                                             | ✅        |
| QA checks                                                                     | ✅        |
| Code format checks                                                            | ✅        |
| Connector package install                                                     | ✅        |
| Build source-zendesk-support docker image for platform linux/x86_64           | ✅        |
| Unit tests                                                                    | ✅        |
| Acceptance tests                                                              | ❌        |

🔗 [View the logs here](https://storage.googleapis.com/airbyte-ci-reports/airbyte-ci/connectors/test/pull_request/ddavydov_16887-source-zendesk-support-add-skills-based-routing-streams/1687807139/27783b08acf5af802a6c549701c5113b388802b7/source-zendesk-support/0.5.0/output.html)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/tools/ci_connector_ops/ci_connector_ops/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=source-zendesk-support test
```

**Comment by octavia-squidington-iii** [comment]
##  source-zendesk-support test report (commit [`eb0b7899e6`](https://github.com/airbytehq/airbyte/pull/27450/commits/eb0b7899e6147de6fb24fb4baa38414e1783a95d)) - ❌

⏲️  Total pipeline duration: 1008 seconds

| Step                                                                          | Result   |
|:------------------------------------------------------------------------------|:---------|
| Validate airbyte-integrations/connectors/source-zendesk-support/metadata.yaml | ✅        |
| Connector version semver check                                                | ✅        |
| Connector version increment check                                             | ✅        |
| QA checks                                                                     | ✅        |
| Code format checks                                                            | ❌        |
| Connector package install                                                     | ✅        |
| Build source-zendesk-support docker image for platform linux/x86_64           | ✅        |
| Unit tests                                                                    | ✅        |
| Acceptance tests                                                              | ❌        |

🔗 [View the logs here](https://storage.googleapis.com/airbyte-ci-reports/airbyte-ci/connectors/test/pull_request/ddavydov_16887-source-zendesk-support-add-skills-based-routing-streams/1687853826/eb0b7899e6147de6fb24fb4baa38414e1783a95d/source-zendesk-support/0.6.0/output.html)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/tools/ci_connector_ops/ci_connector_ops/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=source-zendesk-support test
```

**Comment by octavia-squidington-iii** [comment]
##  source-zendesk-support test report (commit [`6feba3ef62`](https://github.com/airbytehq/airbyte/pull/27450/commits/6feba3ef622b1563f3ec31af2d7f450bbd8ea0c9)) - ❌

⏲️  Total pipeline duration: 1965 seconds

| Step                                                                          | Result   |
|:------------------------------------------------------------------------------|:---------|
| Validate airbyte-integrations/connectors/source-zendesk-support/metadata.yaml | ✅        |
| Connector version semver check                                                | ✅        |
| Connector version increment check                                             | ✅        |
| QA checks                                                                     | ✅        |
| Code format checks                                                            | ✅        |
| Connector package install                                                     | ✅        |
| Build source-zendesk-support docker image for platform linux/x86_64           | ✅        |
| Unit tests                                                                    | ✅        |
| Acceptance tests                                                              | ❌        |

🔗 [View the logs here](https://storage.googleapis.com/airbyte-ci-reports-multi/airbyte-ci/connectors/test/pull_request/ddavydov_16887-source-zendesk-support-add-skills-based-routing-streams/1687857785/6feba3ef622b1563f3ec31af2d7f450bbd8ea0c9/source-zendesk-support/0.6.0/output.html)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/tools/ci_connector_ops/ci_connector_ops/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=source-zendesk-support test
```

**Comment by octavia-squidington-iii** [comment]
##  source-zendesk-support test report (commit [`8eac91c935`](https://github.com/airbytehq/airbyte/pull/27450/commits/8eac91c93500ab869859fa4a8736c2a043c2ad1e)) - ❌

⏲️  Total pipeline duration: 980 seconds

| Step                                                                          | Result   |
|:------------------------------------------------------------------------------|:---------|
| Validate airbyte-integrations/connectors/source-zendesk-support/metadata.yaml | ✅        |
| Connector version semver check                                                | ✅        |
| Connector version increment check                                             | ✅        |
| QA checks                                                                     | ✅        |
| Code format checks                                                            | ✅        |
| Connector package install                                                     | ✅        |
| Build source-zendesk-support docker image for platform linux/x86_64           | ✅        |
| Unit tests                                                                    | ✅        |
| Acceptance tests                                                              | ❌        |

🔗 [View the logs here](https://storage.googleapis.com/airbyte-ci-reports-multi/airbyte-ci/connectors/test/pull_request/ddavydov_16887-source-zendesk-support-add-skills-based-routing-streams/1687860878/8eac91c93500ab869859fa4a8736c2a043c2ad1e/source-zendesk-support/0.6.0/output.html)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/tools/ci_connector_ops/ci_connector_ops/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=source-zendesk-support test
```

**Comment by octavia-squidington-iii** [comment]
##  source-zendesk-support test report (commit [`81e689d754`](https://github.com/airbytehq/airbyte/pull/27450/commits/81e689d75455ec2a5dc034ad569b349ac6215aeb)) - ✅

⏲️  Total pipeline duration: 1674 seconds

| Step                                                                          | Result   |
|:------------------------------------------------------------------------------|:---------|
| Validate airbyte-integrations/connectors/source-zendesk-support/metadata.yaml | ✅        |
| Connector version semver check                                                | ✅        |
| Connector version increment check                                             | ✅        |
| QA checks                                                                     | ✅        |
| Code format checks                                                            | ✅        |
| Connector package install                                                     | ✅        |
| Build source-zendesk-support docker image for platform linux/x86_64           | ✅        |
| Unit tests                                                                    | ✅        |
| Acceptance tests                                                              | ✅        |

🔗 [View the logs here](https://storage.googleapis.com/airbyte-ci-reports-multi/airbyte-ci/connectors/test/pull_request/ddavydov_16887-source-zendesk-support-add-skills-based-routing-streams/1687863925/81e689d75455ec2a5dc034ad569b349ac6215aeb/source-zendesk-support/0.6.0/output.html)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/tools/ci_connector_ops/ci_connector_ops/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=source-zendesk-support test
```

**Pull request merged**